src/Microsoft.Identity.Client/Cache/CacheFallbackOperations.cs

```diff
@@ -42,49 +42,8 @@ internal static class CacheFallbackOperations
         internal /* internal for testing only */ const string DifferentAuthorityError = 
             "Not expecting authority to have a different env than the RT and IdT";
 
-        public static void WriteMsalRefreshToken(ITokenCacheAccessor tokenCacheAccessor,
```

Comment:
not used?

Reply:
Yes, this code had zero references.  It's only used within ADAL to write an MSAL refresh token for forwards/compat between the caches.

---
In reply to: [241515228](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/723#discussion_r241515228) [](ancestors = 241515228)